modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/actions/PreIssueAccessTokenClientCredentialsGrantTestCase.java

```diff
@@ -0,0 +1,354 @@
+package org.wso2.identity.integration.test.actions;
```

Comment:
Add license header

Reply:
Updated with the commit 3f3b2671fc4aac10f5292af1606f409f6b30b2e9